### PR TITLE
Arch Linux support

### PR DESCRIPTION
**Pull request by sempervictus**
### Description

* Adds Arch Linux support to the postgresql cookbook
* Reduces the Chef 13+ restriction (tested against 12.x).

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable

### CODE REVIEW

**Comment by kitchen-porter** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please include a <a href="https://github.com/sous-chefs/line-cookbook/blob/master/CHANGELOG.md">CHANGELOG</a> entry.</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This Pull Request is probably missing tests.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: metadata.rb [hunk 1]
```diff
@@ -8,8 +8,8 @@
 version           '7.1.3'
 source_url        'https://github.com/sous-chefs/postgresql'
 issues_url        'https://github.com/sous-chefs/postgresql/issues'
-chef_version      '>= 13.8'
+chef_version      '>=12.20'
```
Comment (Xorima) [inline metadata.rb]: Please can this be reverted back to supporting chef 13
```suggestion
chef_version      '>=13.8'
```

**Comment by sempervictus** [comment]
Sure, but it already does as 13>12

**Comment by damacus** [comment]
Hi @sempervictus can you open a new PR with just the Arch support please?

**Pull request closed without merging**